.huskyrc.js

```diff
@@ -6,6 +6,6 @@
 
 module.exports = {
   hooks: {
-    "pre-commit": "yarn lint-staged"
+    "pre-commit": "yarn run lint-staged"
```

Comment:
i don't think this necessary. usually `npm run` used. but yarn can deal with it as it was before...
but maybe i don't know something - then tell me ;)

Reply:
couldnt commit without that on my system